internal/services/producer.go

```diff
@@ -36,13 +38,14 @@ func NewProducer(config *config.CanaryConfig) Service {
 
 func (p *Producer) Start() {
 	log.Printf("Starting producer")
-	// TODO: define the actual payload based on the agreed JSON schema
-	payload := "payload"
 	go func() {
 		for {
+			timestamp := time.Now().UnixNano() / 1000000 // timestamp in milliseconds
```

Comment:
We should probably UTC this to prevent timezone complications down the road. `Now()` is going to yield server's local time.

Reply:
But 

> UnixNano returns t as a Unix time, the number of nanoseconds elapsed since January 1, 1970 UTC

So things like DST changes shouldn't affect the value of `timestamp` right?

Reply:
Yes, indeed. Were you thinking about something different @jeremyary ?

Reply:
No, Tom's right there, I glazed over the epoch implication and didn't look up the method. Disregard :)